Cargo.toml

```diff
@@ -25,6 +25,17 @@ base64 = "^0.12"
 url = { version = "^2.1", features = ["serde"] }
 snafu = "^0.6"
 
+#[dependencies.snafu]
```

Comment:
In case of `no_std` env, you need to import snafu like below (no need to include `guide` feature)